From 5ccc622bd4569e88429f14c6e260e19292be8aa6 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 29 Jun 2024 11:59:16 -0700
Subject: [PATCH] Docs/mm/damon/index: add a link to admin-guide doc

Readers of DAMON subsystem document would want to further learn how they
can use DAMON from user-space.  Provide the link to the admin guide.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/index.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/mm/damon/index.rst b/Documentation/mm/damon/index.rst
index 3d3b345d8889..f524f2d4a2eb 100644
--- a/Documentation/mm/damon/index.rst
+++ b/Documentation/mm/damon/index.rst
@@ -34,3 +34,6 @@ configurations.
    design
    api
    maintainer-profile
+
+To utilize and control DAMON from the user-space, please refer to the
+administration :doc:`guide </admin-guide/mm/damon/index>`.
-- 
2.39.2

